tests/testthat/test-workflow_set.R

```diff
@@ -129,6 +129,80 @@ test_that("creating workflow sets", {
   expect_true(is.null(car_set_4$result[[2]]))
 })
 
+test_that("workflow_set can handle case weights", {
+   lr_spec <- linear_reg() %>% set_engine("lm")
+   knn_spec <- nearest_neighbor() %>%
+      set_engine("kknn") %>%
+      set_mode("regression")
+
+   cars <-
+      mtcars %>%
+      dplyr::mutate(
+         wts = hardhat::importance_weights(runif(nrow(.))),
+         non_wts = runif(nrow(.))
+      )
+
+   # do it all right:
+   expect_silent({
+      car_set_1 <-
+         workflow_set(
+            list(reg = mpg ~ ., nonlin = mpg ~ wt + 1 / sqrt(disp)),
+            list(lm = lr_spec),
+            case_weights = wts
+         ) %>%
+         workflow_map(
+            "fit_resamples",
+            resamples = vfold_cv(cars, v = 5)
+         )
+   })
+
+   expect_true(has_case_weights(car_set_1$info[[1]]$workflow[[1]]))
+
+   # specify a column that is not case weights:
```

Comment:
We discussed removing the workflow sets specific validation checks. I think these tests should still live on, but they should be checking that workflows is giving a meaningful error message for these cases in the `$.notes` column of the resulting object.

I think that this and the following tests in this `test_that()` block should also be converted to individual tests themselves. The comment attached to each `expect_` test should become the test description